pallets/protos/src/lib.rs

```diff
@@ -426,7 +429,7 @@ pub mod pallet {
 			tags: BoundedVec<BoundedVec<u8, <T as pallet::Config>::StringLimit>, T::MaxTags>,
 			linked_asset: Option<LinkedAsset>,
 			license: UsageLicense<T::AccountId>,
-			_cluster: Option<Cluster<T::AccountId>>,
+			cluster: Option<Cluster<T::AccountId>>,
```

Comment:
Err, isn't that storing the whole big structure?

Reply:
Changed it to store only the cluster_id, i.e. `Hash128`.

Reply:
Btw, otherwise made no sense, it would copy the data and leave it into a detached state. You didn't consider that?